vasern/src/core/vasern-queryable/Queryable.d.ts

```diff
@@ -0,0 +1,25 @@
+import { NewObject } from './../../plugins/vasern-objectid/ObjectID.d';
+
+export default class Queryable {
+    constructor(data: NewObject);
+
+    data(): NewObject;
+
+    get(query: Object | string): NewObject | undefined;
+
+    find(query: Object | string): NewObject | undefined;
+
+    filter(query: Object | Function): Queryable;
```

Comment:
I just added `Function` as an option for query. For example, we can filter as one of following:

```js
Document.filter({ name: "John" });

// or 
Document.filter(item => item.name == "John" && item.age === 23)
```

Reply:
Nice! I was thinking of how awesome this would be and now it's there. Good job :+1: